vrk-kpa/suomidigi
# SUOPA-692 Minievent form

To test: 

- make fresh
- go to: https://suomidigi.docker.sh/en/form/add-new-event 
- open to another window: https://xd.adobe.com/spec/ba71d258-eba5-4a06-57b8-e3b89749d908-cc40/screen/a554ee30-51ef-4b5c-b9f4-4a5e2a77c95d/Suomidigi-minitapahtuma-luonti/ 
- compare these two pages and make sure that they match visually. The page https://suomidigi.docker.sh/en/form/add-new-event  should look like the design in https://xd.adobe.com/spec/ba71d258-eba5-4a06-57b8-e3b89749d908-cc40/screen/a554ee30-51ef-4b5c-b9f4-4a5e2a77c95d/Suomidigi-minitapahtuma-luonti/ 
- check also that the start date and time and end date and time fields work properly (you can choose the date and time and they show in the input fields fully)
